### PR TITLE
Update monerod stagenet url 

### PR DESCRIPTION
**Pull request by delta1**
## Description

Updated the stagenet url to the one which is suggested on [monerodocs](https://monerodocs.org/infrastructure/networks/ )

## Motivation and Context

Previous url stopped working 

## How Has This Been Tested?

http://monero-stagenet.exan.tech:38081/get_height 

## Checklist:

* [x] I'm merging against the `tari-script` branch.
* [ ] I have squashed my commits into a single commit.

### CODE REVIEW

**Comment by StriderDM** [comment]
We had previously moved away from this server due to it running an outdated version of Monero. I am not sure if it has been updated since then.

There may be a larger issue here since community.xmr.to (of which the current public server being used is part of) and stagenet.xmr-tw.org are also down.

**Comment by delta1** [comment]
xmr.to has shutdown, please suggest a better alternative if you have one

**Comment by hansieodendaal** [comment]
I could not get this new configuration to work properly with testing; I suspect the current stagenet wallet address we use may be the problem:
``` rust
2021-07-02 07:52:08.749844100 [tari::application] INFO  Tari Merge Mining Proxy (0.8.11-8d34066730833cd5f383e6b1ac9a041629b11887-release)
2021-07-02 07:54:27.405929400 [tari_mm_proxy::proxy] DEBUG request: {"id":1,"jsonrpc":"2.0","method":"getblocktemplate","params":
  {"wallet_address":"55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt","extra_nonce":
  "3f3c9d841a875dda"}} (connection=close,content-length=211,content-type=application/json,host=127.0.0.1,user-agent=XMRig/6.12.1 
  (Windows NT 10.0; Win64; x64) libuv/1.41.0 msvc/2019) 
2021-07-02 07:54:27.406157700 [tari_mm_proxy::proxy] DEBUG [monerod] request: POST http://monero-stagenet.exan.tech:38081/json_rpc 
2021-07-02 07:54:27.408468200 [reqwest::connect] DEBUG starting new connection: http://monero-stagenet.exan.tech:38081/
2021-07-02 07:54:28.215799900 [reqwest::async_impl::client] DEBUG response '400 Bad Request' for http://monero-stagenet.exan.tech:38081/json_rpc
2021-07-02 07:54:28.215983400 [tari_mm_proxy::proxy] ERROR Error handling request: Failed to send request to monerod: 
  error decoding response body: expected value at line 1 column 1 
```

**Comment by delta1** [comment]
Thanks for the feedback @hansieodendaal that is helpful

**Comment by delta1** [comment]
Closing to resolve this first

File: integration_tests/helpers/mergeMiningProxyClient.js [hunk 1]
```diff
@@ -12,19 +12,24 @@ class MergeMiningProxyClient {
...
+        id: "0",
+        method: "getblocktemplate",
+        params: {
+          wallet_address:
+            "5AUoj81i63cBUbiKY5jybsZXRDYb9CppmSjiZXC8ZYT6HZH6ebsQvBecYfRKDYoyzKF2uML9FKkTAc7nJvHKdoDYQEeteRW",
+          reserve_size: 60,
+        },
+      });
+      console.log(res.data);
+      console.log("Blocktemplate:", res.data.result.blocktemplate_blob);
```
Comment (delta1) [inline integration_tests/helpers/mergeMiningProxyClient.js]: need to comment these out again

**Review by StriderDM — APPROVED** [review]

**Pull request merged**